### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by recampbell**

### CODE REVIEW

**Comment by recampbell** [comment]
@reviewbybees

File: src/main/resources/index.jelly [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <div>
-  Allows you to login to Jenkins with a Google account and restrict access to a Google Apps Domain.
+  Allows you to log in to Jenkins with a Google account and restrict access to a Google Apps Domain.
```
Comment (jglick) [inline src/main/resources/index.jelly]: Isn’t this a bit pedantic? But :+1:

**Comment by jglick** [comment]
OK, :+1: assuming the tests still pass!

**Comment by recampbell** [comment]
Don't you think this is good enough @jglick ?

**Pull request merged**